### PR TITLE
about, beerstyles and beerdetails pages/footer styles/end point for beerstyles/beer data

### PR DESCRIPTION
**Pull request by ameyakoshti**
final1

### CODE REVIEW

**Comment by ameyakoshti** [comment]
@jmcriffey @ananavati @aweary

File: client/components/beercard.jsx [hunk 1]
```diff
@@ -4,10 +4,16 @@ import FlatButton from "material-ui/FlatButton";
 import Divider from "material-ui/Divider";
 import Beer from "../images/beer-icon-240.png";
 
+const styles = {
+  card: {
+    width: "250px"
+  }
+}
+
 export class BeerCard extends React.Component {
```
Comment (jeffrifwald) [inline client/components/beercard.jsx:13]: Can we name this file `beer-card.jsx` to follow naming conventions?

File: client/components/beerdetails.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
```
Comment (jeffrifwald) [inline client/components/beerdetails.jsx]: Same here, `beer-details.jsx`.

File: client/components/beerstyle.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
```
Comment (jeffrifwald) [inline client/components/beerstyle.jsx]: This one too, `beer-style.jsx`.

**Comment by jeffrifwald** [comment]
Looking good. I just had a few comments about file naming conventions.

**Comment by ananavati** [comment]
👍  same file-name comments as @jmcriffey

**Comment by ameyakoshti** [comment]
File names updated.

**Pull request merged**